lib/discourse_updates.rb

```diff
@@ -150,10 +150,28 @@ def new_features
         end
       return nil if entries.nil?
 
+      entries.map! do |item|
+        next item if !item["experiment_setting"]
+
+        item["experiment_setting"] = nil if !SiteSetting.respond_to?(item["experiment_setting"]) ||
+          ![false, true].include?(SiteSetting.send(item["experiment_setting"]))
+        item
+      end
+
       entries.select! do |item|
         begin
-          item["discourse_version"].nil? ||
-            Discourse.has_needed_version?(current_version, item["discourse_version"])
+          valid_version =
+            item["discourse_version"].nil? ||
+              Discourse.has_needed_version?(current_version, item["discourse_version"])
+
```

Comment:
Don't display feature if plugin is not available. I wonder if plugin should be enabled or just installed. Decided that just installed is enough because new feature announcement may convince someone to enable.

Reply:
Yes I think this is fair enough :+1: